lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -1655,6 +1678,17 @@ instance Arbitrary (PostTransactionData t) where
         <*> arbitrary
         <*> arbitrary
 
+instance Arbitrary (ForgeTokenData t) where
+  arbitrary = ForgeTokenData
+        <$> arbitrary
+        <*> (ApiT <$> genTokenNameSmallRange)
+        <*> arbitrary
+        <*> arbitrary
+        <*> arbitrary
+
+instance Arbitrary addr => Arbitrary (AddressForgeAmount addr) where
+  arbitrary = AddressForgeAmount <$> arbitrary <*> arbitrary
```

Comment:
Suggestion: use the `applyArbitraryX` family of functions to reduce boilerplate.

```suggestion
    arbitrary = applyArbitrary2 AddressForgeAmount
```